polaris-react/src/components/Filters/components/FilterPill/tests/FilterPill.test.tsx

```diff
@@ -83,18 +85,22 @@ describe('<Filters />', () => {
     });
 
     it('when not selected, renders a caret icon', () => {
-      const wrapper = mountWithApp(<FilterPill {...defaultProps} />);
+      const wrapper = mountWithApp(<FilterPill {...defaultProps} />, {
+        features: {polarisSummerEditions2023: true},
+      });
       expect(wrapper.findAll(UnstyledButton)[0]).toContainReactComponent(Icon, {
-        source: CaretDownMinor,
+        source: ChevronDownMinor,
       });
     });
 
     it('when selected, does not render a caret icon', () => {
```

Comment:
```suggestion
    it('when selected, does not render a chevron icon', () => {
```
Not sure if this update is worth it but thought I'd still mention it.